torch/csrc/jit/passes/peephole.cpp

```diff
@@ -206,6 +208,16 @@ void PeepholeOptimizeImpl(Block* block, bool addmm_fusion_enabled) {
             node->input()->debugName());
         node->output()->replaceAllUsesWith(input_node->input());
       }
+    } else if (getProfilingMode() &&
```

Comment:
Why is `getProfilingMode` here? Isn't this a valid transform all the time?

Reply:
it should be valid. I was trying to minimize the leaking from profiling-related related stuff.